src/pdf/DriverOverview/DriverOverviewPdf.tsx

```diff
@@ -18,12 +20,16 @@ export interface DriverOverviewTableData {
     instructions?: string;
     clientIsActive: boolean;
     numberOfLabels: number;
+    collection_centre: {
+        name: string;
+        isDelivery: boolean;
+    };
 }
 
```

Comment:
That's fair, I still think it might be clearer to take out the fields out of the `collection_centre` structure - `isDelivery` doesn't make much sense to me under `collection_centre`
Unless there's a reason why they have to be that way?

Reply:
Oh wait you've done that haven't you, silly github not showing me changes on the comment >:(